nibabel/nicom/ascconv.py

```diff
@@ -0,0 +1,207 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Parse the "ASCCONV" meta data format found in a variety of Siemens MR files.
+"""
+import re
+import ast
+from ..externals import OrderedDict
```

Comment:
I suspect this is included for compatibility, but this has been in the stdlib for all supported versions of Python:

```suggestion
from collections import OrderedDict
```

Reply:
Actually - now we require Python 3.6 - aren't all dicts OrderedDicts?

Reply:
```suggestion
```